x-pack/plugins/fleet/common/services/generate_new_agent_policy.ts

```diff
@@ -21,7 +21,6 @@ export function generateNewAgentPolicyWithDefaults(
     namespace: 'default',
     monitoring_enabled: Object.values(dataTypes),
     inactivity_timeout: TWO_WEEKS_SECONDS,
-    is_protected: false,
```

Comment:
same question here

Reply:
same as above - removing `is_protected` from the `NewAgentPolicy` type